src/redux/books/books.js

```diff
@@ -0,0 +1,37 @@
+const initialState = [];
+
+// Action_types
+
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+// Reducer
+
+export default function booksReducer(state = initialState, action) {
+  switch (action.type) {
+    case ADD_BOOK:
+      return {};
+    case REMOVE_BOOK:
+      return {};
+    default:
+      return state;
+  }
```

Comment:
- [ ] Kindly implement these reducer for the different action types.

  - The ADD_BOOK action should add a book to the store:

    ```
    case ADD_BOOK:
      return [
        {
          id: action.id,
          title: action.title,
          author: action.author
        },
        ...state,
      ];
    ```

  -  The REMOVE_BOOK should remove a book from the store based on the ID

      ```
      case REMOVE_BOOK:
        return state.filter((book) => book.id !== action.id);
      ```